modules/tensor_mechanics/src/materials/RecomputeRadialReturn.C

```diff
@@ -140,6 +140,18 @@ RecomputeRadialReturn::getIsotropicShearModulus()
   return shear_modulus;
 }
 
+Real
+RecomputeRadialReturn::getIsotropicBulkModulus()
+{
+  Real shear_modulus = getIsotropicShearModulus();
+  Real dil_modulus = _elasticity_tensor[_qp](0,0,0,0);
```

Comment:
I'm pretty sure Ben would disagree with the idea of putting a comment to explain the equation, but a comment would be a nice way to make the name clear. With three letters it is sometimes hard to tell when the name is an abbreviation and when it is an acronym when looking at the code for the first time -- at least that's been my experience with migrating some of the older material models -- so I am pushing for more verbose code/comments now.  Thank you for considering this!